contentcuration/contentcuration/frontend/channelEdit/views/progress/ProgressBar.vue

```diff
@@ -8,14 +8,19 @@
       <Icon v-else-if="isDone" large color="greenSuccess" data-test="success">
         check_circle
       </Icon>
-      <VProgressCircular v-else indeterminate color="primary" />
+      <VProgressCircular
+        v-else
+        :indeterminate="!task"
+        :progress="progress"
+        :color="$themeTokens.loading"
+      />
     </VFlex>
     <VFlex xs11 class="px-4">
       <div v-if="text" class="body-1">
         {{ text }}
       </div>
       <!-- Show progress bar if progress is tracked -->
-      <VLayout v-if="progressPercent !== -1 && !isDone" row align-center class="mt-3">
+      <VLayout v-if="showLinear && progressPercent !== -1 && !isDone" row align-center class="mt-3">
```

Comment:
If the intention is to show the indeterminate spinner only, might need to just make the horizontal progress bar indeterminate

Reply:
It shows an indeterminate spinner when we don't have a taskId to track progress off, and a determinate one when it does. Might need to cross check with you and @jtamiace about how the progress should be implemented - she had specified a circular loader in her designs.

Reply:
Have just created a small separate component for this instead.